conda/install.py

```diff
@@ -56,7 +56,9 @@ def __enter__(self):
         def __exit__(self, exc_type, exc_value, traceback):
             pass
 
-on_win = bool(sys.platform == 'win32')
+from conda.utils import win_path_to_unix
```

Comment:
Please see the comment at the top of this module.  This module needs to be standalone.